src/components/BookDetails/BookDetails.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './BookDetails.css';
+
+export default function BookDetails({
+	coverImg,
+	authors,
+	title,
+	publishDate,
+	description,
+	categories,
+}) {
+	return (
+		<div className="book-details-container day-theme">
+			<img src={`../images/${coverImg}`} alt={title} />
+			<div className="book-details">
+				<p className="book-title">{title}</p>
+				<p className="book-author">
+					By {authors} ({publishDate})
+				</p>
+				<p className="book-genre">
+					<span>{categories}</span>
```

Comment:
Nitpick: Do we need a span if all the text is in that span? I think we could probably just style the `.book-genre` element instead.